docs/about/whats_new.rst

```diff
@@ -11,6 +11,8 @@ v1.8.next
 v1.8.10 (30 January 2023)
 =========================
 
+- Update conda create environment README (:pull:`1394`)
```

Comment:
Wrong place!  Needs to be a few lines up, in the `1.8.next` section.

Reply:
thanks, fixed